packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -256,36 +246,36 @@ export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
     ) : null;
   }
 
-  private _onDragStart(item?: any, itemIndex?: number, selectedItems?: any[], event?: MouseEvent): void {
+  private _onDragStart = (item?: any, itemIndex?: number, selectedItems?: any[], event?: MouseEvent): void => {
     const classNames = this._classNames;
     if (itemIndex) {
       this._updateHeaderDragInfo(itemIndex);
-      this._root.current.classList.add(classNames.borderWhileDragging);
+      (this._root.current as HTMLElement).classList.add(classNames.borderWhileDragging);
       this._async.setTimeout(() => {
         if (this._root.current) {
           this._root.current.classList.add(classNames.noBorderWhileDragging);
         }
       }, CLASSNAME_ADD_INTERVAL);
     }
-  }
+  };
 
-  private _onDragEnd(item?: any, event?: MouseEvent): void {
+  private _onDragEnd = (item?: any, event?: MouseEvent): void => {
     const classNames = this._classNames;
     if (event) {
       this._updateHeaderDragInfo(-1, event);
     }
-    this._root.current.classList.remove(classNames.borderWhileDragging);
-    this._root.current.classList.remove(classNames.noBorderWhileDragging);
-  }
+    (this._root.current as HTMLElement).classList.remove(classNames.borderWhileDragging);
```

Comment:
Just curious why this cast here? Is it because of the type tightening on the root ref above? Why not check on the existence of the `this._root.current` as done on line 255?

Reply:
> Just curious why this cast here? Is it because of the type tightening on the root ref above?

Yes, it is due to `_root` no longer being `any`. The existing logic was not gated with `undefined` checks such as `if (this._root.current)` for when the ref is not assigned, whenever that may be (cc: @arkogupta @laxmikankanala).

We can add those checks, since otherwise this would run-time error when accessing `classList` of `undefined, but I consider that work separate from what I've done here as it modifies existing logic.